### PR TITLE
refactor:support inspecting envelope

### PR DESCRIPTION
**Pull request by chloeyin**

### CODE REVIEW

**Review by patrickzheng200 — APPROVED** [review]
LGTM

File: internal/envelope/envelope.go [hunk 1]
```diff
@@ -1,7 +1,31 @@
...
 // Supported envelope format.
 const (
 	Cose = "cose"
 	Jws  = "jws"
 )
+
+// SpeculateSignatureEnvelopeFormat speculates envelope format by looping all builtin envelope format.
+//
+// TODO: find a better way to inspect the type of envelope.
+// TODO: support inspecting cose format
```
Comment (binbin-li) [inline internal/envelope/envelope.go:20]: Doesn't it support cose right now?

Comment (chloeyin) [reply internal/envelope/envelope.go]: > Doesn't it support cose right now?

Yes, comment is outdated. I removed it.

**Review by binbin-li — APPROVED** [review]
lgtm

**Pull request merged**